master/buildbot/status/buildstep.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, parent, master, step_number, step_type):
         self.hidden = False
         self.logs = []
         self.urls = {}
+        self.artifactUrls = {}
```

Comment:
I like just 'artifacts', it short and clear, moreover it contains not only 'url' but also 'name'. And in future might be configured with additional data that can bring again the renaming issue. 

The less vague it would be in format { artifacts: [ {url: url1, name: name1}, {url: url2, name: name2} ]} - but I